apps/cursor-pro/src/app/(content)/events/_components/create-new-event-form.tsx

```
@@ -0,0 +1,351 @@
+										/>
+										<FormField
+											control={form.control}
+											name={`events.${index}.startsAt`}
+											render={({ field }) => (
+												<FormItem>
+													<FormLabel>Starts at</FormLabel>
+													<DateTimePicker
+														{...field}
+														aria-label="Starts At"
+														value={
+															!!field.value
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)